yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/check/RequestedThirdPartyIdentityFraudOneCheck.java

```diff
@@ -0,0 +1,41 @@
+package com.yoti.api.client.docs.session.create.check;
+
+import com.yoti.api.client.docs.DocScanConstants;
+
+public class RequestedThirdPartyIdentityFraudOneCheck extends RequestedCheck<RequestedThirdPartyIdentityFraudOneConfig> {
```

Comment:
`THIRD_PARTY_IDENTITY_FRAUD_1` checks cannot be requested like regular checks (they are part of the Identity Profile flow).  This isn't needed :)